torch/utils/data/dataloader.py

```diff
@@ -142,6 +142,7 @@ def __init__(self, dataset, batch_size=1, shuffle=False, sampler=None,
                  batch_sampler=None, num_workers=0, collate_fn=default_collate,
                  pin_memory=False, drop_last=False, timeout=0,
                  worker_init_fn=None):
+        torch._C._log_api_usage_once("python.data_loader")
```

Comment:
Well those are _not_ free, so why are we hardcoding them in the core even though literally no other user than FB cares about this feature?

Reply:
i think it's time to actually flesh out a design and expose logging properly. i think this feature would be pretty useful to a wider set of folks -- anyone doing thousands of experiments fleet-wide.

Reply:
I'm ok with adding a API to do this, but some of the decisions taken in this one seem fairly arbitrary to me.